integrations/gitlab/scaGitLabScript.py

```diff
@@ -199,19 +206,44 @@ def to_hungarian_case(input_string):
     hungarian_case_words = input_string[0].upper() + input_string[1:].lower()
     return hungarian_case_words
 
+def str_to_bool(value):
+    return value.lower() in ("true", "1", "yes", "on")
 
 
 if __name__ == "__main__":
 
-    if len(sys.argv) == 1:
-        print("A JSON file name argument must be provided.")
-        sys.exit()
```

Comment:
Not sure but is it okay to remove this given the new logic? this was previously making sure that if no args were provided, the user would know they have to include a JSON file name argument. but now i think we just print an error? unsure what that error looks like

Reply:
I fixed that. If no parameter is passed, then the message is:
`Usage: scaGitLabScript.py <json_file> [--lowering-unreachable true|false]`